api/lib/application/pole-emploi/index.js

```diff
@@ -76,7 +76,7 @@ exports.register = async function (server) {
       path: '/api/pole-emploi/auth-url',
       config: {
         auth: false,
-        handler: PoleEmploiController.getAuthUrl,
+        handler: PoleEmploiController.getAuthenticationUrl,
```

Comment:
```suggestion
        handler: poleEmploiController.getAuthenticationUrl,
```

On dirait qu'on importe deux fois le même `poleEmploiController`, une fois en minuscule, une fois en majuscule…

Reply:
corrigé